wordpress-mobile/WordPress-Android
# Gutenberg - Starter Page Templates Tooltip

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2169

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/2216
`Gutenberg PR` -> https://github.com/WordPress/gutenberg/pull/21974
`WordPress iOS PR` -> https://github.com/wordpress-mobile/WordPress-iOS/pull/14045

To test:

- Download and install the build.
- Open the app and go to Pages.
- Tap on the add a new page button.
- **Expect** to see the templates picker with the tooltip on top.
- Scroll through the template options.
- **Expect** the tooltip to stay visible.
- Tap on the title of the page.
- **Expect** to see the tooltip fade out.
- Close the app.
- Open the app and go to Pages.
- Tap on the add a new page button.
- **Expect** to see the templates picker **without** the tooltip.

PR submission checklist:
